kubernetes/windows/setup.ps1

```diff
@@ -67,9 +67,9 @@ Write-Host ('Extracting Certificate Generator Package')
     Expand-Archive -Path /opt/amalogswindows/certificategenerator.zip -Destination /opt/amalogswindows/certgenerator/ -Force
 Write-Host ('Finished Extracting Certificate Generator Package')
 
-Write-Host ('Installing Windows Azure Monitor Agent');
+$windowsazuremonitoragent = [System.Environment]::GetEnvironmentVariable('WINDOWS_AMA_URL')
```

Comment:
I think, we should validate the url here and fail the build if this is not valid url.

Reply:
The build will fail if the url is not valid and the download doesn't work. What other kind of validation you want to add?

Reply:
> The build will fail if the url is not valid and the download doesn't work. What other kind of validation you want to add?

> if build able to catch then we are good.